### ISSUE TITLE
Guitar Tab notation

### ISSUE THREAD

**Issue by kbrand5333** [w3c/mnx#63]
Is there going to be any sort of functionality for guitar notation, particularly with complex tab?  Music XML is rather limited in how it captures Hal Leonard's tab notation and therefore is not terribly useful to us in this area right now.

**Comment by mdgood** [comment]
Yes, tab is intended for MNX. The current spec focuses on the structural differences between MNX and MusicXML, and so far I don't think we have gotten into many of the details of how tab representations may differ.

Would you be willing to write up the issues you have with MusicXML tab in separate issues so we can address them individually? The main problems that I know about that are specific to MusicXML, rather than the limitations of software implementations, are:

- Bends in MusicXML are represented quite differently than bends in much notation software, making it difficult for software to transfer them accurately.
- Many techniques such as vibrato and palm mute are represented by generic elements such as `<bracket>` rather than more semantic elements. Similarly, playing techniques are underrepresented in MusicXML. This makes it difficult to capture many guitar directions in a semantic way that can guide playback as well as appearance.

The more we can identify these and other tab issues for MNX up front, the better we will be able to address them.

**Comment by kbrand5333** [comment]
I'll work on this when I have some time (not sure when that will be though).  I know I have a list of issues somewhere.

**Comment by snakebyte69** [comment]
Hey Michael I have commented on these issues in other forums before. The problem at the moment is the lack of Notation programs using the MusicXML 3.1 standard. I'm working with Arobas (Guitar Pro) to improve this situation. Notion's version of MusicXML is the most dated. Dorico is yet to implement Guitar TAB.
